src/elements/kw-share-stats/kw-share-stats.html

```diff
@@ -0,0 +1,93 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout.html">
+<link rel="import" href="../kano-style/color.html">
+<dom-module id="kw-share-stats">
+    <template>
+        <style>
+             :host {
+                display: block;
+            }
+            
+            .stats {
+                @apply --layout-horizontal;
+                color: #435055;
+                font-size: 12px;
+                height: 25px;
+            }
+            
+            .stats .likes,
+            .stats .comments,
+            .stats .shares {
+                height: 30px;
+                border: none;
+                background-color: white;
+                color: var(--color-grey-darker);
+                cursor: pointer;
+            }
+            
+            .stats .likes:focus,
+            .stats .comments:focus,
+            .stats .shares:focus {
+                outline: none;
+            }
+            
+            .stats .likes span,
+            .stats .comments span,
+            .stats .shares span {
+                padding-right: 16px;
+            }
+            
+            .stats .likes span img,
+            .stats .comments span img,
+            .stats .shares span img {
+                padding-right: 6px;
+            }
+            
+            div.stats[tombstone] .likes,
+            div.stats[tombstone] .comments,
+            div.stats[tombstone] .shares {
+                width: 80px;
+                height: 20px;
+                background: rgba(0, 0, 0, 0.1);
+                color: transparent;
+                margin-right: 9px;
+            }
+            
+            div.stats[tombstone] .likes span img,
+            div.stats[tombstone] .comments span img,
+            div.stats[tombstone] .shares span img {
+                display: none;
+            }
+        </style>
+        <div class="stats">
+            <button class="likes" on-tap="doLikeAction">
```

Comment:
A method name matching the actual event and not the action would be more fitting something like `_onLikeTapped`